pyvista/core/pointset.py

```diff
@@ -106,29 +106,84 @@ class PolyData(_vtk.vtkPolyData, PointSet, PolyDataFilters):
     - Using vertices and faces
     - From a file
 
+    Parameters
+    ----------
+    var_inp : vtk.PolyData, iterable, optional
+        Flexible input type.  Can be a ``vtk.PolyData``, in which case
+        this PolyData object will be copied if ``deep=True`` and will
+        be a shallow copy if ``deep=False``.
+
+        Otherwise, this must be a points array or list containing one
+        or more points.  Each point must contain 3 dimensions.
+
+    faces : iterable, optional
+        Face connectivity array.  Faces must contain padding
+        indicating the number of points in the face.  For example, the
+        two faces ``[10, 11, 12]`` and ``[20, 21, 22, 23]`` will be
+        represented as ``[3, 10, 11, 12, 4, 20, 21, 22, 23]``.  This
+        lets you have an arbitrary number of points per face.
+
+        When not including the face connectivity array, each point
+        will be assigned to a single vertex.  This is used for point
+        clouds that have no connectivity.
+
+    n_faces : int, optional
+        Number of faces in the ``faces`` connectivity array.  While
+        optional, setting this speeds up the creation of the
+        ``PolyData``.
+
+    lines : iterable, optional
+        The line connectivity array.  Like ``faces``, this array
+        requires padding indicating the number of points in a line
+        segment.  For example, the two line segments ``[0, 1]`` and
+        ``[1, 2, 3, 4]`` will be represented as
+        ``[2, 0, 1, 4, 1, 2, 3, 4]``
+
+    n_lines : int, optional
+        Number of lines in the ``lines`` connectivity array.  While
+        optional, setting this speeds up the creation of the
+        ``PolyData``.
+
+    deep : bool, optional
+        Optional to copy the inputs, or to create a mesh from them
+        without copying them.  Setting ``deep=True`` ensures that the
+        original arrays can be modified outside the mesh without
+        affecting the mesh.
+
     Examples
     --------
-    >>> import pyvista
-    >>> from pyvista import examples
     >>> import vtk
     >>> import numpy as np
+    >>> from pyvista import examples
+    >>> import pyvista
+
+    Create an empty mesh
 
-    >>> surf = pyvista.PolyData()  # Create an empty mesh
+    >>> mesh = pyvista.PolyData()
+
+    Initialize from a vtk.vtkPolyData object
```

Comment:
Here and at the start of the class docstring?
```suggestion
    Initialize from a ``vtk.vtkPolyData`` object
```